server/views/pages/index.njk

```diff
@@ -9,22 +9,30 @@
       <div class="govuk-width-container">
           <h1 class="govuk-heading-l">Access eForms Records</h1>
 
-          {% set introText = 'You can generate reports or search for an eForm record' if isReportingAllowed else 'You can search for eForm records' %}
-
-          <p class="govuk-body govuk-!-margin-bottom-7">{{ introText }}</p>
-
           {% if isReportingAllowed %}
+              <p class="govuk-body">
+                  You can generate reports or search for an eFormm record.
+              </p>
               <p class="govuk-body">
                   <a href="/generate-report" class="govuk-link govuk-link--no-visited-state">Generate eForm reports</a>
               </p>
+              <p class="govuk-body">
+                  <a href="/search-eform" class="govuk-link govuk-link--no-visited-state">View eForm records</a>
+              </p>
+          {% elseif isProviderReportingAllowed %}
```

Comment:
This should be an `if`, as I am in both the reporting and provider reporting groups and I am not seeing the provider reports link. You should be able to see either link or both according to your groups.